plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/config/CubridUserManager.java

```diff
@@ -0,0 +1,119 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.ui.config;
+
+import java.util.List;
+import java.util.Map;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridDataSource;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridPrivilage;
+import org.jkiss.dbeaver.ext.generic.model.GenericStructContainer;
+import org.jkiss.dbeaver.model.DBPDataSource;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.edit.DBECommandContext;
+import org.jkiss.dbeaver.model.edit.DBEPersistAction;
+import org.jkiss.dbeaver.model.exec.DBCExecutionContext;
+import org.jkiss.dbeaver.model.impl.edit.SQLDatabasePersistAction;
+import org.jkiss.dbeaver.model.impl.sql.edit.SQLObjectEditor;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.sql.SQLUtils;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+import org.jkiss.dbeaver.model.struct.cache.DBSObjectCache;
+import org.jkiss.utils.CommonUtils;
+
+public class CubridUserManager extends SQLObjectEditor<CubridPrivilage, GenericStructContainer> /*implements DBEObjectRenamer<OracleSchema>*/ {
+
+    @Override
+    public long getMakerOptions(DBPDataSource dataSource) {
+        return FEATURE_EDITOR_ON_CREATE;
+    }
+
+    @Nullable
+    @Override
+    public DBSObjectCache<? extends DBSObject, CubridPrivilage> getObjectsCache(CubridPrivilage object) {
+        DBSObject parentObject = object.getParentObject();
+        if (parentObject instanceof CubridDataSource container) {
+            return container.getDataSource().getCubridPrivilageCache();
+        }
+        return null;
+    }
+
+    @Override
+    protected CubridPrivilage createDatabaseObject(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBECommandContext context,
+            @NotNull final Object container,
+            @Nullable Object copyFrom,
+            @NotNull Map<String, Object> options) {
+        
+        CubridPrivilage user = new CubridPrivilage((CubridDataSource) container, "NEW_USER", null);
+        return user;
+    }
+    
+    
+
+    @Override
+    protected void addObjectCreateActions(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBCExecutionContext executionContext,
+            @NotNull List<DBEPersistAction> actions,
+            @NotNull ObjectCreateCommand command,
+            @NotNull Map<String, Object> options) {
+        CubridPrivilage user = (CubridPrivilage) command.getObject();
+        StringBuilder builder = new StringBuilder();
+        builder.append("CREATE USER " + this.getUserName(user, command.getProperties()));
```

Comment:
Please use `StringBuilder#append` instead of manual string concatenation. Fix it everywhere in your PR.